avocado/utils/kernel.py

```diff
@@ -22,7 +22,7 @@
 import os
 import shutil
 import tempfile
-from distutils.version import LooseVersion  # pylint: disable=E0611
+from packaging import version as versobjc
```

Comment:
we use [isort](https://pycqa.github.io/isort/) for checking the ordering of the imports and it is complaining about this you have to change this.

Reply:
Cool. I would make this change happen
.